jobs/periodic/fetch/utils/verification/index.ts

```diff
@@ -13,7 +13,7 @@ export function generateToken(): string {
         .toString("base64")
         .replace(/\//g, "-")
         .replace(/\+/g, "_");
-    logger.debug("Generated token: ", token);

```

Comment:
Uh nice, logging credentials 😅  ... This function will hopefully be removed soon, so no need to fix ...

Reply:
😅  I haven't even noticed